src/foam/nanos/auth/PMAuthService.js

```diff
@@ -0,0 +1,213 @@
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'PMAuthService',
+  extends: 'foam.nanos.auth.ProxyAuthService',
+
+  implements: [
+    'foam.nanos.auth.EnabledAware',
+    'foam.nanos.boot.NSpecAware'
```

Comment:
Unlike DAO's where there can be more than one, I don't think this is necessary. You can just hardcode the prefix name as "auth".